app/boards/shields/levinson/Kconfig.defconfig

```diff
@@ -0,0 +1,17 @@
+#Copyright (c) 2021 The ZMK Contributors
+#SPDX-License-Identifier: MIT
```

Comment:
```suggestion
# Copyright (c) 2021 The ZMK Contributors
# SPDX-License-Identifier: MIT
```

Reply:
@Nicell thanks for the further instructions, I have added the Levinson as an option in the files you mentioned following the formatting of all the other shields. I hope this was correct. I also added in those spaces in the copyright lines. Not sure how that one got missed ;)